content/en/borealis/Reference/ref-deployment-file.md

```diff
@@ -41,15 +41,15 @@ targets:
     strategy: <strategyName>
 # The list of manifests sources
 manifests:
-  # A directory containing multiple manifests. Instructs Borealis to read all yaml|yml files in the directory and deploy all manifests to the target defined in    `targets`.
+  # A directory containing multiple manifests. Borealis all yaml|yml files in the directory and deploy all manifests to the target defined in    `targets`.
```

Comment:
I'm not sure if this is 100% understandable

Reply:
Haha, it is 

```suggestion
  # A directory containing multiple manifests. Borealis reads all yaml|yml files in the directory and deploys the manifests to the target defined in `targets`.
```